backend/drive/drive.go

```diff
@@ -691,6 +712,9 @@ func (f *Fs) shouldRetry(ctx context.Context, err error) (bool, error) {
 	}
 	switch gerr := err.(type) {
 	case *googleapi.Error:
+		if gerr.Code == 400 { // Retry on Invalid Value
```

Comment:
Retrying 400 errors is a recipe for infinite loops on parameter errors.

Can you be more specific?